content/deploy/install/services.md

```diff
@@ -353,6 +353,91 @@ supergraphCompose:
 $ helm upgrade --install router -f router.yaml {{< param application_name >}}/router -n {{< param application_name >}}
 ```
 
+## Optional: Calendar Service (Draft)
+
+The {{< param brand_name >}} calendar service monitors work calendar definitions and creates work calendar entries in real time, both in the [Graph DB](#db) and Timeseries DB.
+
+> **Requirements:** The calendar service requires the [GraphDB](#db), [Keycloak](#keycloak), and [NATS](#nats) services.
+
+> **Note:** It is a prerequisite to have a Timeseries DB installed such as [InfluxDB](https://influxdata.com/), [QuestDB](https://questdb.io) or [TimescaleDB](https://www.timescale.com/). The following instructions are specific to QuestDB.
```

Comment:
```suggestion
{{% notice "note" %}}
It is a prerequisite to have a time-series DB installed such as [InfluxDB](https://influxdata.com/), [QuestDB](https://questdb.io) or [TimescaleDB](https://www.timescale.com/). The following instructions are specific to QuestDB.
{{% /notice %}}
```

Reply:
Shortcodes and other stuff like this are documented in [CONTRIBUTING.MD](https://github.com/libremfg/rhize-docs/blob/main/CONTRIBUTING.md#notices)